lua/nvim-tree.lua

```diff
@@ -364,15 +364,19 @@ local function setup_autocommands(opts)
   create_nvim_tree_autocmd("BufReadPost", {
     callback = function()
       if filters.config.filter_no_buffer or renderer.config.highlight_opened_files then
```

Comment:
`highlight_opened_files` has a default of "none".

We should test for that too.

Reply:
4fc74ca resolves this